doc/reference/reference_lua/box_space/create_index.rst

```diff
@@ -361,50 +368,61 @@ Creating an index using the path option for map fields (JSON-path indexes)
 --------------------------------------------------------------------------------
 
 To create an index for a field that is a map (a path string and a scalar value),
-specify the path string during index_create, that is,
-:code:`parts={` :samp:`{field-number},'{data-type}',path = '{path-name}'` :code:`}`.
-The index type must be ``'tree'`` or ``'hash'`` and the field's contents
+specify the path string during index creation, like this:
+
+..  cssclass:: highlight
+..  parsed-literal::
+
+    :extsamp:`parts = {{*{field-number}*}, {*{'data-type'}*}, path = {*{'path-name'}*}}`
+
+The index type must be ``tree`` or ``hash`` and the contents of the field
```

Comment:
Sometimes we call it `tree`, in other cases — TREE. Is there a difference?

Reply:
no, they are equal